cmd/start.go

```diff
@@ -83,23 +86,22 @@ var startCmd = &cobra.Command{
 		} else {
 			level = zapcore.InfoLevel
 		}
-		l, err := logger.NewZapLogger(level)
+		l, err := logger.NewZapLogger(level, viper.GetString(logFormatFlagName))
 		if err != nil {
 			log.Fatalf("can't initialize zap logger: %v", err)
 		}
 		logger := logger.NewLogger(l, Debug)
 		rtLogger := logger.WithFields(zap.String("component", "start"))
 
 		if viper.GetString(syncProviderFlagName) != "" {
-			rtLogger.Warn("DEPRECATED: The --sync-provider flag has been deprecated. " +
-				"Docs: https://github.com/open-feature/flagd/blob/main/docs/configuration/configuration.md")
+			rtLogger.Warn("DEPRECATED: the --sync-provider flag has been deprecated " +
+				"Docs: https://github.com/open-feature/flagd/blob/main/docs/configuration.md")
```

Comment:
This link as been broken, should be `https://github.com/open-feature/flagd/blob/main/docs/configuration/configuration.md`